scripts/ci/pre_commit/check_system_tests.py

```diff
@@ -34,7 +34,7 @@
 
 errors: list[str] = []
 
-WATCHER_APPEND_INSTRUCTION = "list(dag.tasks) >> watcher()"
```

Comment:
Hmm. I think it's als used as instruction part (see below) so possibly "checking" should be separated from "instruction" - because the insstruction should still work the same